.circleci/config.yml

```diff
@@ -167,7 +130,10 @@ jobs:
             - node-v1-{{ checksum "package-lock.json" }}
       - run:
           name: Install Node JS dependencies
-          command: npm install
+          command: |
+            git config --global url."https://github.com/".insteadOf git@github.com:
+            git config --global url."https://".insteadOf git://
```

Comment:
Do we need run these two commands when working locally as well? If so can it also be added to the README